source/CardGames.py

```diff
@@ -636,7 +645,37 @@ def fugitive_repeating_turn(self):
                       #Functions to check for illegally placed hideouts and burns would go here
       elif turn == "pass":
         print("\nPassing...")
-      return NewHideouts
+    
+    def marshall_repeating_turn(self):
+      #Obtain user's deck choice
+      while True:
+        try:
+          deck = int(input("\nSelect which deck to draw two cards from ['1' (1-14), '2' (15-28), '3' (29-42)] and press enter... "))
+        except ValueError:
+          print("Invalid input, try again.")
+          continue
+        if deck in [1, 2, 3]:
+          break
+      
+      match deck:
```

Comment:
Woah this is fancy! Looks great, also I really like the changes to the print outs. Good stuff

Reply:
Thank you brother! I appreciate it.